modules/nodes-containers-projected-volumes-creating.adoc

```diff
@@ -126,11 +128,12 @@ metadata:
 type: Opaque
 ----
 
-. Create a pod configuration file similar to the following that includes a `volumes` section:
+. Create a pod with a projected volume.
```

Comment:
Make sure steps end in a colon, per IBM style guide: https://www.ibm.com/docs/en/ibm-style?topic=format-procedures
Also, there's an "a" but not a "b," so maybe this should be rolled up into step 2?